external-crates/move/crates/move-analyzer/editors/code/src/context.ts

```diff
@@ -2,22 +2,55 @@
 // Copyright (c) The Move Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import type { Configuration } from './configuration';
-import { lint, inlayHintsForType } from './configuration';
+import {
+    MOVE_CONF_NAME, LINT_OPT, TYPE_HINTS_OPT,
+    SUI_PATH_OPT, SERVER_PATH_OPT, Configuration,
+} from './configuration';
+import * as childProcess from 'child_process';
 import * as vscode from 'vscode';
 import * as lc from 'vscode-languageclient/node';
+import * as semver from 'semver';
 import { log } from './log';
-import { sync as commandExistsSync } from 'command-exists';
+import { assert } from 'console';
 import { IndentAction } from 'vscode';
 
+function semanticVersion(path: string, args?: readonly string[]): semver.SemVer | null {
+    const versionString = childProcess.spawnSync(
+        path, args, { encoding: 'utf8' },
+    );
+    if (versionString.stdout) {
+        // Version string looks as follows: 'COMMAND_NAME SEMVER-SHA'
+        const versionStringWords = versionString.stdout.split(' ');
```

Comment:
You may want to set the split limit as well, to make sure you get at most two parts?